npm/ipsm/ipsm_test.go

```diff
@@ -818,12 +855,15 @@ func TestIPSetCannotBeAddedAsElementDoesNotExist(t *testing.T) {
 */
 func TestMain(m *testing.M) {
 	metrics.InitializeAll()
-	ipsMgr := NewIpsetManager()
-	ipsMgr.Save(util.IpsetConfigFile)
```

Comment:
Do we need "Save" and "Restore" anymore to store existing ipsets and restore them after completion of UTs when UTs do not use "fexec"?

Reply:
when using fexec we don't need to save/restore because no state is created